### DISCUSSION TITLE
PS acv

### DISCUSSION THREAD

**Discussion by Yakonrus** [dlemstra/Magick.NET#951] [Help]
Hello.
Is it possible to use color correction from Photoshop?
Curves preset .acv

**Comment by dlemstra** [comment]
I don't know what that is. Maybe you should also ask your question here: https://github.com/ImageMagick/ImageMagick/discussions?

**Reply by Yakonrus** [reply]
Thx

**Reply by Yakonrus** [reply]
![изображение](https://user-images.githubusercontent.com/4978662/121813830-78cf7080-cc76-11eb-9167-5b243bd694b6.png)
